sdk/mediaservices/Azure.Media.Analytics.Edge/README.md

```diff
@@ -0,0 +1,194 @@
+# Azure Live Video Analytics for IoT Edge client library for .NET
+
+Live Video Analytics on IoT Edge provides a platform to build intelligent video applications that span the edge and the cloud. The platform offers the capability to capture, record, and analyze live video along with publishing the results, video and video analytics, to Azure services in the cloud or the edge. It is designed to be an extensible platform, enabling you to connect different video analysis edge modules (such as Cognitive services containers, custom edge modules built by you with open-source machine learning models or custom models trained with your own data) to it and use them to analyze live video without worrying about the complexity of building and running a live video pipeline.
+
+Use the client library for Live Video Analytics on IoT Edge to:
+
+- Simplify interactions with the [Microsoft Azure IoT SDKs](https://github.com/azure/azure-iot-sdks) 
+- Programatically construct media graph topologies and instances
+
+[Product documentation][doc_product] | [Direct methods][doc_direct_methods] | [Media graphs][doc_media_graph] | [Source code][source] | [Samples][samples]
+
+## Getting started
+### Authenticate the client
+n/a
```

Comment:
Why N/A? Doesn't the service require authentication?

Reply:
There is no client for this service

Reply:
Your samples show a serviceclient instance. How do you use the service without a client?

Reply:
That serviceclient is coming from another sdk, the azure-iot-hub sdk. This sdk just has models and passes those models onto the azure-iot hub sdk

Reply:
the Edge SDK works on connection string. we can add that info the readme. but all that details are well documented on the IoT sdk readme. may be we should just link to that readme.

Reply:
The goal of the readme is to get a customer who knows nothing about the service started. It has to be clear how to connect to the service and execute hello world scenarios.